samples/Extensions/Table/TableFunction.cs

```diff
@@ -4,28 +4,26 @@
 using System;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
-using Newtonsoft.Json.Linq;
 
 namespace SampleApp
 {
     public static class TableFunction
     {
         [Function("TableFunction")]
-        [TableOutput("outputQueue", Connection = "ServiceBusConnection")]
+        [TableOutput("OutputTable", Connection = "AzureWebJobsStorage")]
```

Comment:
Is there a specific reason we need to change the connection string to the default?

@ankitkumarr was the original name something you were trying to have as a sample?

Reply:
For me this did look not intended but more like reused code from a service bus example. It had **"ServiceBusConnection"** as the configuration key name and a ``TableOutput`` attribute with the table name that is **"queue"**. 

_There is a choice not specifying the Connection at all but doing so does show that it can be pointed to any other storage connection than the default._

Reply:
Indeed! Had overlooked the name/trigger mismatch.